automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -23,7 +24,7 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
 
   "launch a simple workflow" in withWebDriver { implicit driver =>
     val user = Config.Users.owner
-    implicit val authToken: AuthToken = AuthToken(user)
+    implicit val authToken: AuthToken = UserAuthToken(user)
```

Comment:
I kinda wish `Credentials` had a function like:
```
  def makeAuthToken(): AuthToken = UserAuthToken(this)
```
so that client code wouldn't have to care about how it's obtained. If it had been there, the number of changes in this PR would have been reduced. Adding it now would be roughly the same number of changes now but fewer changes later (likely never) but would also be cleaner. I leave it up to you whether or not to do it.

Reply:
sure